### PR TITLE
Changelog editorial prior to 1.10.0 release

### PR DESCRIPTION
**Pull request by abernix**
Just some editorial polish to get out ahead of the release.

### CODE REVIEW

File: NEXT_CHANGELOG.md [hunk 1]
```diff
@@ -34,81 +34,87 @@ By [@Geal](https://github.com/geal) in https://github.com/apollographql/router/p
...
 
-Adds new option called `max_age` that is used like this:
+Adds new option called `max_age` to the existing `cors` object which will set the value returned in the [`Access-Control-Max-Age`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age) header. As was the case previously, when this value is not set **no** value is returned.
+
+It can be enabled using our standard time notation, as follows:
+
 ```
 cors:
-  max_age: 86400secs
+  max_age: 1 day
```
Comment (BrynCooke) [inline NEXT_CHANGELOG.md:43]: Just checking does this work? I've not seen it with a space between the numerals and the suffix before.
https://docs.rs/humantime/latest/humantime/fn.parse_duration.html

Comment (abernix) [reply NEXT_CHANGELOG.md:43]: I guess I don't know.  I assumed sanity would presume, but I'll change it back to without the space. 😄

Comment (abernix) [inline NEXT_CHANGELOG.md:43]:
```suggestion
  max_age: 1day
```

Comment (o0Ignition0o) [reply NEXT_CHANGELOG.md]: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=d519d5e04c32d12fd94adf55fa42b065 all good!

Comment (o0Ignition0o) [reply NEXT_CHANGELOG.md]: ah came just too late 😅

Comment (abernix) [reply NEXT_CHANGELOG.md]: thank you for checking! good to know.

**Review by BrynCooke — APPROVED** [review]

**Pull request merged**